src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -53,10 +112,10 @@
                                         ;; have to do this by ID not ref, since the fields are generated within Table
                                         (after-update #(.focus (.getElementById js/document "focus"))))}]])
            [table/Table
```

Comment:
could we make the font in this table bigger to match that of dataset attributes?